### ISSUE TITLE
Improve regex highlighting in Python

### ISSUE THREAD

**Issue by monosans** [Binaryify/OneDark-Pro#678]
**Describe what you want**

The regex highlighting in Python in `Dark+` is fancier than in `One Dark Pro` in Python.

**Additional context**

As you can see in the screenshots, `Dark+` uses 3 colors for regex highlighting, while `One Dark Pro` only uses 2.

![Dark+](https://user-images.githubusercontent.com/76561516/149808828-40c6aa16-0b72-4fb9-bfcd-affd6e0e811f.png)
![One Dark Pro](https://user-images.githubusercontent.com/76561516/149808823-3ff927a4-225c-4e73-b381-ce7904dcc284.png)

Code from screenshots for reproducing:

```python
import re
tag_pattern = re.compile(r"\[(id\d+?)\|.+?\]")
```

**Comment by Binaryify** [comment]
plz check 3.13.19~

**Comment by monosans** [comment]
> plz check 3.13.19~

Looks good. Thank you.

**Comment by tomaciazek** [comment]
Hey, it looks like the related commit has introduced configuration that overrides pre-existing block:
```typescript
{
  name: 'Escape Characters',
  scope: 'constant.character.escape',
  settings: {
    foreground: 'fountainBlue',
  },
},
```

This changes highlighting for tokens outside of regular expressions. Was that intended?

**Comment by Binaryify** [comment]
@tomaciazek no, just want improve regex highlighting, could you provide some code to show what highlight had been affected？

**Comment by tomaciazek** [comment]
Sure.
In Python:
![image](https://user-images.githubusercontent.com/15631478/154441430-92e1ea01-a8ae-43be-bcb1-e01de6130c53.png)

In JSON:
![image](https://user-images.githubusercontent.com/15631478/154441468-39a1b692-a36a-4e9b-9960-f7b25d9a3b0e.png)

**Comment by Binaryify** [comment]
@tomaciazek plz check v3.13.20~

**Comment by tomaciazek** [comment]
Now there's good old fountain-blue on escape characters :) Thanks!

**Comment by prot23** [comment]
> **Describe what you want**
> 
> The regex highlighting in Python in `Dark+` is fancier than in `One Dark Pro`.
> 
> **Additional context**
> 
> As you can see in the screenshots, `Dark+` uses 3 colors for regex highlighting, while `One Dark Pro` only uses 2.
> 
> ![Dark+](https://user-images.githubusercontent.com/76561516/149808828-40c6aa16-0b72-4fb9-bfcd-affd6e0e811f.png) ![One Dark Pro](https://user-images.githubusercontent.com/76561516/149808823-3ff927a4-225c-4e73-b381-ce7904dcc284.png)
> 
> Code from screenshots for reproducing:
> 
> ```python
> import re
> tag_pattern = re.compile(r"\[(id\d+?)\|.+?\]")
> ```
#596

**Comment by prot23** [comment]
import re
tag_pattern = re.compile(r"\[(id\d+?)\|.+?\]")

**Comment by prot23** [comment]
`import re
tag_pattern = re.compile(r"\[(id\d+?)\|.+?\]")`

**Comment by prot23** [comment]
`import re
tag_pattern = re.compile(r"\[(id\d+?)\|.+?\]")`

**Comment by LeaReXx** [comment]
One Dark Pro interferes with RTL text in HTML and reverses HTML tags, Fix it
![image](https://user-images.githubusercontent.com/88275860/161503528-70c278a5-123e-47a2-89fc-605eda443587.png)
